### ISSUE TITLE
Support operation on multiple data frames, for example concat, merge, join, append, compare, etc

### ISSUE THREAD

**Issue by shuangquanhuang** [gventuri/pandas-ai#86]
### 🚀 The feature

Support operation on multiple data frames, for example concat, merge, join, append, compare, etc

### Motivation, pitch

Support operation on multiple data frames, for example concat, merge, join, append, compare, etc

**Comment by gventuri** [comment]
@shuangquanhuang super nice addition, will work on it!

**Comment by ambujpawar** [comment]
Hey, @gventuri I can work on it. However, before starting with the issue. I have a few questions:
- How do you propose to handle this? Because currently the user queries like ```pandas_ai.run(df, prompt='Which are the 5 happiest countries?')```
- Should it be handle databases like ```pandas_ai.run(df1, df2, prompt='Merge the two databases?')```

Or do you have something else in mind?

**Comment by gventuri** [comment]
@ambujpawar maybe we could also accept a list in the first param and then conditionally run the workflow on only 1 df vs multiple dfs.

Something like this:
```
pandas_ai.run(df1, prompt='Which are the 5 happiest countries?')
pandas_ai.run([df1, df2], prompt='Which are the 5 happiest countries?')
```

What do you think?

**Comment by ambujpawar** [comment]
I dont have a strong preference for any. But out of the these two options, I prefer:
```
pandas_ai.run(df1, df2, prompt='Which are the 5 happiest countries?')
```

Because to be it seems to be more "easy" for the user as well as more intuitive. So shall I go ahead with that?

**Comment by mzy2240** [comment]
The current version does not understand groupby-related operation, which might also be good to have.

**Comment by HenriqueAJNB** [comment]
This groupby tasks should be another issue. Can you create that @mzy2240?

**Comment by gventuri** [comment]
@mzy2240 @HenriqueAJNB I've created one (#127 )

**Comment by gventuri** [comment]
Feature added with this commit: https://github.com/gventuri/pandas-ai/commit/be4c717a9b0f07b88ee2aed729b637413df28a51

Closing
